src/duct/logger/timbre.clj

```diff
@@ -39,4 +39,12 @@
                    {:config config, :?ns-str ns-str, :?file file, :?line line}))))
 
 (defmethod ig/init-key :duct.logger/timbre [_ config]
-  (->TimbreLogger config))
+  (->TimbreLogger (let [prev-root timbre/*config*]
+                    (if (::set-config! config true)
+                      (do (timbre/set-config! config)
+                          (assoc config ::prev-root-config prev-root))
```

Comment:
> The :duct.logger.timbre/set-config! key should be something more like: :set-root-binding?

To :bike: :house: names some more another alternative might be `::global-logging-config?` depends whether you want to favour clojure specific terminology or user intent.

Happy to go with whichever you prefer though...